### PR TITLE
tests: Remove unused import statements

### PR DESCRIPTION
**Pull request by Techievena**
Remove the unused imports in tests directory,
to help prepare for coala/coala#3066

Fixes https://github.com/coala/coala-bears/issues/1126

### CODE REVIEW

**Comment by jayvdb** [comment]
Do not use the GitHub rebase feature.  See https://github.com/coala/coala-bears/pull/1143/commits
Please rebase properly.  Come onto Gitter for help fixing your branch.

File: tests/python/test_files/pylint_test.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import sys
-import platform  # Unused import
```
Comment (jayvdb) [inline tests/python/test_files/pylint_test.py]: I think everything in ` tests/python/test_files/` shouldnt be modified - they are intentionally bad syntax

File: tests/natural_language/LanguageToolBearTest.py [hunk 2]
```diff
@@ -6,13 +6,6 @@
 from tests.BearTestHelper import generate_skip_decorator
 from tests.LocalBearTestHelper import verify_local_bear
 
-try:
```
Comment (jayvdb) [inline tests/natural_language/LanguageToolBearTest.py]: this small block needs to stay here.  See the voodoo in https://github.com/coala/coala-bears/pull/1135/files to work around this problem.

ie. use

```
    import language_check
    import guess_language
    language_check
    guess_language
```

File: tests/python/test_files/pylint_test.py (hunk 1 above)
Comment (Techievena) [inline tests/python/test_files/pylint_test.py]: Sorry @jayvdb I am unable to figure out what changes I have to make in this

Comment (jayvdb) [inline tests/python/test_files/pylint_test.py]: Restore this line as it was.  We'll find a solution for this line in https://github.com/coala/coala/issues/3066 , after your patch is merged.

Comment (jayvdb) [inline tests/python/test_files/pylint_test.py]: You havent restored this line exactly.

Comment (Techievena) [inline tests/python/test_files/pylint_test.py]: Sorry I'd do that I thought It'd be same if I rewrite the line

File: tests/natural_language/LanguageToolBearTest.py [hunk 3]
```diff
@@ -9,10 +9,11 @@
 try:
     import language_check
     import guess_language
+    language_check
```
Comment (abhsag24) [inline tests/natural_language/LanguageToolBearTest.py]: you had to remove these right? not just the import keyword but everything?

**Comment by abhsag24** [comment]
@Techievena you didnt remove this: `tests/python/test_files/pylint_test.py:2: 'platform' imported but unused`

**Comment by Techievena** [comment]
@abhsag24 I didn't remove [tests/python/test_files/pylint_test.py:2: 'platform' imported but unused] as @jayvdb commented it as a bad syntax and it will be handled in a different issue. So he asked me to restore it.

File: tests/natural_language/LanguageToolBearTest.py (hunk 3 above)
Comment (sils) [inline tests/natural_language/LanguageToolBearTest.py]: use a coala ignore statement here?

**Comment by sils** [comment]
@jayvdb what about https://github.com/coala/coala-bears/pull/1143/commits/eceeb35b7c8e085d2b81010ec5824c98e9836920#r92918230 ?

File: tests/natural_language/LanguageToolBearTest.py (hunk 3 above)
Comment (jayvdb) [inline tests/natural_language/LanguageToolBearTest.py]: this approach adds two lines, and works with pyflakes natively.
your solution adds or amends two lines, and only works within coala.
I'm not seeing the benefit, especially not work holding up this patch and potentially conflicting with other patches incoming.  If necessary we can switch later.

**Comment by jayvdb** [comment]
ack eceeb35

**Comment by jayvdb** [comment]
I think it needs a rebase, but I cant verify that easily atm

**Comment by sils** [comment]
yep it does

**Comment by Techievena** [comment]
Do I have to modify this pull request??? (Rebase it????)

**Comment by jayvdb** [comment]
Yup. rebase it pls

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/1143/commits/ff1721015cf5a2dedba603ace62f3774424ed860

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
@Techievena @jayvdb is there any bear we can activate to prevent this from happening in the future?

**Pull request merged**